actgen/test.py

```diff
@@ -0,0 +1,121 @@
+import argparse
+import logging
+import random
+
+import numpy as np
+import gym
+import seeding
+import torch
+from tqdm import tqdm
+
+from . import utils
+from . import wrappers as wrap
+from .agents import RandomAgent, DQNAgent
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Trial:
+    def __init__(self, test=True):
+        args = self.parse_args()
+        self.params = self.load_hyperparams(args)
+        # hyper params to set up testing
+        self.params['max_env_steps'] = 10000
+        self.params['eval_every_n_steps'] = 100
+        self.params['epsilon_final'] = 0
+        self.params['n_eval_episodes'] = 10
+        self.params['replay_warmup_steps'] = 0
+        if self.params['test'] or test:
```

Comment:
The only thing I'd use test mode for here is to skip loading the q network. That way we can make sure the rest of the script runs in the CI system, but we don't need to save the models in git